pom.xml

```diff
@@ -96,6 +96,7 @@
         <ehcache.version>2.10.9.2</ehcache.version>
         <greenmail.version>2.0.0-alpha-2</greenmail.version>
         <httpclient.version>4.5.3</httpclient.version>
+        <httpclient5.version>5.1.4</httpclient5.version>
```

Comment:
Why not use the latest 5.2.1 version?

Reply:
Cause it's the version currently managed by spring-boot... Upgraded to 5.1.2.